docs/source/system_reference_guide/oss_documentation/doi_and_licensing.ipynb

```diff
@@ -0,0 +1,99 @@
+{
```

Comment:
<p>Should we include a small statement about why you would choose one over the other?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/357/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>Like Apache 2.0 over MIT or something like that? Or just why those three in general are good options?</p>

Reply:
this is what a chatbot came up with:

> MIT License:
> 
> It’s a short and simple permissive license.
> It allows you to use, copy, modify, merge, publish, distribute, sublicense, and sell copies of the software without many restrictions.
> It’s most similar to the 2-clause BSD license.
> 
> BSD 3-Clause License:
> 
> It’s similar to the MIT license but adds a term that prevents someone from claiming false endorsement.
> This non-endorsement clause prohibits promotion of any derived work using the name of the license or its authors.
> It’s compatible with GPLv2 and v3.
> 
> Apache 2.0 License:
> 
> It requires you to keep the license file, the NOTICE file if there is one, and show notice for modified files.
> It addresses some patent-related issues, so companies use it a lot.
> It’s compatible with GPLv3 but not v2 (due to the patent clauses).
> If developers desire the assurance of patent protection, Apache 2.0 is the better choice.



but this is really just a nice to have since users can click on the license links and read more

Reply:
Cool, that helps! I'll look into adding short statements about each one, thanks!

Reply:
<p>I would send people to <a href="https://choosealicense.com" target="_blank">https://choosealicense.com/</a>  and  <a href="https://opensource.org" target="_blank">https://opensource.org</a> </p><p>I'm not sure we can make any statements on the specifics of the licenses.</p>

Reply:
<p><a href="https://choosealicense.com/licenses" target="_blank">https://choosealicense.com/licenses/</a> is a good page. Note Choosealicense is MIT biased.</p>

Reply:
<p>Given that I don't know the difference it may be that other scientists don't and it would be a good addition</p>

Reply:
<p>I agree to all the above comments.. It would be good to have a brief description or table highlighting usage of each license based on a few attributes such as - redistribution, commercial usage, sharing...</p>